<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depret</class>
 <widget class="QDialog" name="depret">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>112</width>
     <height>86</height>
    </rect>
   </property>
   <property name="title">
    <string>              Accion</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QRadioButton" name="retirar">
      <property name="text">
       <string>Retirar</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="depositar">
      <property name="text">
       <string>Depositar</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>200</y>
     <width>79</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Balance Final:</string>
   </property>
  </widget>
  <widget class="QLabel" name="balance_final">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>200</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="balance_actual">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>172</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>41</x>
     <y>172</y>
     <width>87</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Balance Actual:</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>166</x>
     <y>30</y>
     <width>200</width>
     <height>130</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QLineEdit" name="id"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Cuenta</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Clave</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="clave"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="monto"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Monto</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="validar">
      <property name="text">
       <string>validar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
